File: output/ChatGPT/codenet/C++/Java/s684825280.java
import java.util.ArrayList;
import java.util.Collections;
import java.util.List;
import java.util.Scanner;

public class s684825280 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int k = scanner.nextInt();
        List<Integer> a = new ArrayList<>();
        int[] cnt = new int[(int) (1e5 * 2) + 13];

        for (int i = 0; i < n; ++i) {
            int x = scanner.nextInt();
            cnt[x]++;
        }

        for (int i = 0; i < cnt.length; ++i) {
            if (cnt[i] > 0) {
                a.add(cnt[i]);
            }
        }

        int ans = 0;
        Collections.sort(a);
        int m = a.size() - k;

        for (int i = 0; i < m; ++i) {
            ans += a.get(i);
        }

        System.out.println(ans);
    }
}

File: output/ChatGPT/codenet/C++/Python/s684825280.py
import sys
from collections import Counter

n, k = map(int, sys.stdin.readline().split())
cnt = Counter(sys.stdin.readline().split())

a = []
for x in cnt.values():
    a.append(x)

ans = 0
a.sort()
m = len(a) - k
for i in range(m):
    ans += a[i]
    
print(ans)